apps/src/netsim/NetSimVizSimulationNode.js

```diff
@@ -39,6 +39,12 @@ var NetSimVizSimulationNode = module.exports = function (sourceNode,
   this.correspondingNodeID_ = sourceNode.entityID;
 
   /**
+   * UUID of the simulation node that this viz element represents.
```

Comment:
This comment is confusing because it seems like "simulation node" might refer to a NetSimVizSimulationNode, whereas it actually refers to a NetSimNode. It might be easier to read if you just used type names in comments, e.g. by changing "simulation node" to NetSimNode (here and above).